proto/neutron/contractmanager/query.proto

```diff
@@ -46,6 +46,13 @@ message QueryParamsResponse {
 
 // QueryFailuresRequest is request type for the Query/Failures RPC method.
 message QueryFailuresRequest {
+  // address of the contract which Sudo call failed.
+  string address = 1;
+  cosmos.base.query.v1beta1.PageRequest pagination = 3;
+}
+
+// QueryFailureRequest is request type for the Query/Failures RPC method.
+message QueryFailureRequest {
```

Comment:
`pagination` field seems to be not needed logically (and not used) for `QueryFailureRequest`.
Please remove it